mmv1/products/compute/Disk.yaml

```diff
@@ -430,11 +430,20 @@ properties:
     name: 'provisionedIops'
     description: |
       Indicates how many IOPS must be provisioned for the disk.
-      Note: Update currently only supported by hyperdisk skus, allowing for an update of IOPS every 4 hours
+      Note: Update currently only supported by hyperdisk skus, allowing for an update of IOPS every 4 hours without deletion and recreations
```

Comment:
Maybe "To update your disk more frequently, you'll need to manually delete and recreate it"? The current wording could be read to imply Terraform will plan a recreation.